duecredit/tests/test_io.py

```diff
@@ -211,3 +212,10 @@ def test_enumeratedentries():
     assert_raises(KeyError, enumentries.fromrefnr, 666)
 
     assert_equal(entries, sorted(enumentries, key=lambda x: x[1]))
+
+def test_get_text_rendering():
+    citation_bibtex = Citation(_sample_bibtex, path='mypath')
+    citation_doi = Citation(_sample_doi, path='mypath')
+    # smoke test
+    assert_true(get_text_rendering(citation_bibtex))
```

Comment:
ok -- we should talk about patches. I tried but can't seem to make it work. It does nothing. 

```
@patch('duecredit.io.format_bibtex')
def test_get_text_rendering(mocked_format_bibtex):
    from ..io import get_text_rendering
    citation_bibtex = Citation(_sample_bibtex, path='mypath')
    citation_doi = Citation(_sample_doi, path='mypath')
    # smoke test
    assert_true(get_text_rendering(citation_bibtex))
    mocked_format_bibtex.assert_called_with(citation_bibtex.entry)
    assert_true(get_text_rendering(citation_doi))
```
this won't work. hints?